services/src/handlers/wcs.rs

```diff
@@ -349,8 +351,14 @@ async fn get_coverage<C: Context>(
             p,
             query_rect,
             query_ctx,
-            no_data_value,
-            request_spatial_ref,
+            GdalGeoTiffDatasetMetadata {
+                no_data_value,
+                spatial_reference: request_spatial_ref,
+            },
+            GdalGeoTiffOptions {
+                compression_num_threads: get_config_element::<crate::util::config::Gdal>()?.compression_num_threads,
```

Comment:
i think we must be careful with the config elements. We changed most of the config lookup to run only once at server start. There we set the context variables and they are "nerver" read again. In this case there might be no harm in reading it in the handler BUT every call will read the config file and i think that is not a nice solution.

Reply:
Do you have an example for that?

Reply:
we load the config in the server startup and store the values in the context: https://github.com/geo-engine/geoengine/blob/8a8e873ed2824eb608c52fdc1af8ac9ec913b1eb/services/src/pro/server.rs#L115

Reply:
siehe Diskussion in Slack